spec/features/sign_in_feature_spec.rb

```diff
@@ -15,6 +15,25 @@ def expect_successful_sign_in_for(educator)
     it { expect_successful_sign_in_for(pals.west_marcus_teacher) }
   end
 
+  context 'teacher fails signin too many times' do
+    let(:rich_email) { 'rich@demo.studentinsights.org' }
+    let(:rich) { Educator.find_by_email(rich_email) }
+
+    it 'locks out the teacher' do
+      expect(rich.locked_at).to eq nil
+
+      sign_in_attempt(rich_email, 'badpassword')
+      sign_in_attempt(rich_email, 'badpassword')
+      sign_in_attempt(rich_email, 'badpassword')
+      sign_in_attempt(rich_email, 'badpassword')
+      sign_in_attempt(rich_email, 'badpassword')
+
+      rich.reload
+
+      expect(rich.locked_at).not_to eq nil
```

Comment:
maybe `expect(rich.failed_attempts).to eq 5` also?

Reply:
Also, this won't be testing the production path since we're not using the LDAP configuration in dev or test.  Maybe a place where adding a mock LDAP server and swapping that in for this test would help.